### PR TITLE
ARC-815 Removing reposyncstate column

### PR DESCRIPTION
**Pull request by mboudreau**
And removing more `repoSyncState` things in the codebase.  SQS tests are still flaky...

### CODE REVIEW

File: db/migrations/20220112110000-remove-reposyncstate-column.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            "Subscriptions"."repoSyncState"        AS repo_sync_state,
+            "Subscriptions"."selectedRepositories" AS selected_repositories,
+            "Subscriptions"."syncStatus"::text     AS sync_status,
+            "Subscriptions"."syncWarning"          AS sync_warning,
+            EXTRACT(EPOCH FROM current_timestamp-"Subscriptions"."createdAt")/3600  AS hours_since_created_at,
+            EXTRACT(EPOCH FROM current_timestamp-"Subscriptions"."updatedAt")/3600  AS hours_since_updated_at
+      FROM "Subscriptions";
+    `);
+
+
```
Comment (KAbakumovAtl) [inline db/migrations/20220112110000-remove-reposyncstate-column.js:31]: Should  the Installations view be the part of the revert step as well?

Comment (mboudreau) [reply db/migrations/20220112110000-remove-reposyncstate-column.js]: whoops, forgot to push that.

**Review by KAbakumovAtl — APPROVED** [review]

File: test/frontend/util/get-jira-configuration-helpers.test.ts [hunk 2]
```diff
@@ -169,11 +169,6 @@ describe("getInstallation", () => {
 		await RepoSyncState.destroy({ truncate: true });
 	});
 
-	it("Should get installation from repoSyncState", async () => {
-		const result = await getInstallation(GitHubAPI(), subscription);
-		expect(result).toMatchObject(match);
-	});
```
Comment (mboudreau) [inline test/frontend/util/get-jira-configuration-helpers.test.ts:175]: this was a duplicate of the one below.  Must have missed it when I removed the feature flag.

File: src/api/index.ts [hunk 3]
```diff
@@ -139,7 +139,7 @@ router.get("/", (_: Request, res: Response): void => {
 });
 
 router.get(
-	"/:installationId/:jiraHost/repoSyncState.json",
+	"/:installationId/:jiraHost/syncstate",
```
Comment (thombergs) [inline src/api/index.ts:142]: This API is just an Admin API to be used for investigations? Or is it called from somewhere else?

**Review by thombergs — APPROVED** [review]

File: src/api/index.ts (hunk 3 above)
Comment (mboudreau) [reply src/api/index.ts:142]: only admin API, nothing else uses it.  I wanted to remove confusion where possible.

**Pull request merged**